tests/baselines/reference/bundlerDirectoryModule(moduleresolution=bundler).trace.json

```diff
@@ -18,6 +18,136 @@
     "File '/lib/cjs/index.tsx' does not exist.",
     "File '/lib/cjs/index.d.ts' exists - use it as a name resolution result.",
     "======== Module name '../lib' was successfully resolved to '/lib/cjs/index.d.ts'. ========",
+    "======== Resolving module '@typescript/lib-esnext' from '/.src/__lib_node_modules_lookup_lib.esnext.d.ts__.ts'. ========",
```

Comment:
I believe this baseline change to be correct; this test would (outside of the test harness) have an implicit target of `esnext`, but we were forcing it to be es5.